database/seeds/CoordSeeder.php

```diff
@@ -0,0 +1,102 @@
+
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Coords;
+
+
+class CoordSeeder extends Seeder
+{
+
+
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $array = [
+            '[44, 494, 10]', 
+            '[49, 447, 10]', 
+            '[69, 405, 10]',
+            '[93, 321, 10]', 
+            '[93, 365, 10]',
+            '[93, 277, 10]', 
+            '[93, 233, 10]',
+            '[73, 199, 10]', 
+            '[44, 163, 10]', 
+            '[44, 113, 10]',
+            '[57, 70, 10]',
+            '[103, 60, 10]', 
+            '[149, 60, 10]',
+            '[195, 60, 10]', 
+            '[235, 90, 10]',
+            '[240, 135, 10]',
+            '[253, 179, 10]',
+            '[282, 210, 10]', 
+            '[305, 230, 10]', 
+            '[328, 250, 10]',
+            '[353, 270, 10]', 
+            '[383, 270, 10]', 
+            '[413, 270, 10]',
+            '[443, 270, 10]', 
+            '[473, 270, 10]', 
+            '[503, 270, 10]',
+            '[525, 250, 10]', 
+            '[545, 230, 10]', 
+            '[565, 210, 10]',
+            '[585, 190, 10]',
+            '[605, 170, 10]', 
+            '[615, 130, 10]',
+            '[645, 94, 10]',
+            '[697, 63, 10]',
+            '[730, 63, 10]', 
+            '[763, 63, 10]',
+            '[798, 80, 10]',
+            '[801, 119, 10]',
+            '[782, 145, 10]', 
+            '[769, 178, 10]', 
+            '[769, 218, 10]',
+            '[794, 252, 10]', 
+            '[803, 283, 10]', 
+            '[803, 313, 10]',
+            '[803, 343, 10]', 
+            '[792, 385, 10]', 
+            '[789, 416, 10]',
+            '[789, 446, 10]', 
+            '[803, 483, 10]', 
+            '[818, 507, 10]',
+            '[848, 507, 10]',
+            '[875, 484, 10]', 
+            '[875, 444, 10]',
+            '[875, 404, 10]', 
+            '[875, 364, 10]', 
+            '[875, 324, 10]',
+            '[875, 284, 10]', 
+            '[875, 244, 10]', 
+            '[875, 204, 10]',
+            '[875, 164, 10]', 
+            '[875, 124, 10]', 
+            '[875, 84, 10]', 
+            '[918, 63, 10]',
+            '[959, 83, 10]',
+            '[959, 123, 10]',
+            '[959, 163, 10]', 
+            '[959, 203, 10]', 
+            '[959, 243, 10]',
+            '[959, 283, 10]', 
+            '[959, 323, 10]', 
+            '[959, 363, 10]',
+            '[959, 403, 10]', 
+            '[959, 443, 10]', 
+            '[959, 473, 10]', 
+            '[959, 503, 10]' 
+        ];
+
+        foreach($array as $item) {
+            Coords::create([
+                'coords' => $item]);
```

Comment:
This is totally me nitpicking, but I think it looks nicer to have the closing array bracket and parenthesis on their own line. So:
`Coords::create([
   'coords' => $item
]);`